salt/utils/gitfs.py

```diff
@@ -39,6 +39,7 @@
 from salt.config import DEFAULT_HASH_TYPE
 from salt.config import DEFAULT_MASTER_OPTS as _DEFAULT_MASTER_OPTS
 from salt.exceptions import FileserverConfigError, GitLockError, get_error_message
+from salt.grains.core import get_machine_id
```

Comment:
I'd move the whole `get_machine_id` logic to `salt.utils.platform`, `salt.grains.core` is not a utility module...

Reply:
I will change that to use salt.utils.files.get_machine_identifier, this is a line missed from earlier fixes in code development, and should have been changed.  Of course from  platform if go that way